tomviz/ImageStackModel.h

```diff
@@ -0,0 +1,54 @@
+/******************************************************************************
+
+  This source file is part of the tomviz project.
+
+  Copyright Kitware, Inc.
+
+  This source code is released under the New BSD License, (the "License").
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+******************************************************************************/
+
+#ifndef tomvizImageStackModel_h
+#define tomvizImageStackModel_h
+
+#include <QAbstractTableModel>
+#include <QFileInfo>
+#include <QModelIndex>
```

Comment:
Again - a lot of includes, QObject certainly isn't needed, QModelIndex and QString may not be,

Reply:
Fixed in https://github.com/OpenChemistry/tomviz/pull/1544/commits/26c1a89a8118057a1031d374654141b98a2dc5fb